packages/gatsby-source-drupal/src/gatsby-node.js

```diff
@@ -268,143 +273,101 @@ exports.sourceNodes = async (
 
   drupalFetchActivity.start()
 
-  let allData
-  try {
-    const res = await requestQueue.push([
-      urlJoin(baseUrl, apiBase),
-      {
-        username: basicAuth.username,
-        password: basicAuth.password,
-        headers,
-        searchParams: params,
-        responseType: `json`,
-      },
-    ])
-    allData = await Promise.all(
-      _.map(res.body.links, async (url, type) => {
-        const dataArray = []
-        if (disallowedLinkTypes.includes(type)) return
-        if (!url) return
-        if (!type) return
-
-        // Lookup this type in our list of language alterable entities.
-        const isTranslatable = languageConfig.translatableEntities.some(
-          entityType => entityType === type
-        )
+  const listResponse = await worker([
```

Comment:
you should be using the `requestQueue` here and other places

Reply:
Yeah I figured out how to do that yesterday. Feels weird pushing a return value from a queue.push to a queue.push but here we go :)